src/collection-browser.ts

```diff
@@ -1811,6 +1811,20 @@ export class CollectionBrowser
     }
   }
 
+  /**
+   * Returns the input string, but removing one set of quotes from all instances of
+   * ""clauses wrapped in two sets of quotes"". This assumes the quotes are already
+   * URL-encoded.
+   *
+   * This should be a temporary measure to address the fact that the __href__ field
+   * sometimes acquires extra quotation marks during query rewriting. Once there is a
+   * full Lucene parser in place that handles quoted queries correctly, this can likely
+   * be removed.
+   */
+  private collapseRepeatedQuotes(str?: string): string | undefined {
+    return str?.replace(/%22%22(?!%22%22)(.+?)%22%22/g, '%22$1%22');
```

Comment:
q: if it is undefined, will the display bork?

Reply:
The case where `__href__` is undefined should maintain the same behavior as before: leaving it undefined in the tile model, and allow the tile components to decide how to render the tile link.

So if `collapseRepeatedQuotes` receives an undefined input, it's expected that it should output undefined to match that prior behavior.